ksqldb-engine/src/main/java/io/confluent/ksql/planner/plan/QueryFilterNode.java

```diff
@@ -123,6 +126,7 @@ public QueryFilterNode(
     extractKeysAndSystemCols();
 
     // Extraction of lookup constraints
+    this.typeManager = new ExpressionTypeManager(source.getSchema(), metaStore);
```

Comment:
because (smartly so) you're using the column type instead of the literal type (`col.get().type()`) this is no longer used -right?